src/Microsoft.AspNetCore.Razor.Language/BoundAttributeDescriptorExtensions.cs

```diff
@@ -27,5 +27,27 @@ public static bool IsDefaultKind(this BoundAttributeDescriptor attribute)
 
             return string.Equals(attribute.Kind, TagHelperConventions.DefaultKind, StringComparison.Ordinal);
         }
+
+        public static bool ExpectsStringValue(this BoundAttributeDescriptor attribute, string name)
```

Comment:
We able to mark these internal? Don't want to make these guys public unless we absolutely have to.